backend/plugin/db/driver.go

```diff
@@ -247,8 +247,9 @@ type QueryContext struct {
 	// Mainly used for the search path of PostgreSQL.
 	Schema string
 	// Limit is the maximum row count returned. No limit enforced if limit <= 0
-	Limit   int
-	Explain bool
+	Limit     int
+	Explain   bool
+	UserEmail string
```

Comment:
How about define the `Labels map[string]string` here?  `UserEmail` here is some kind of strange..

Reply:
This is only applicable to BigQuery. How about using "User *store.UserMessage" instead of "UserEmail string" here?

Reply:
LGTM, maybe `Operator *store.UserMessage`?

Reply:
Found "import cycle not allowedgo list". Let's just call it "OperatorEmail string" instead?